authors/apps/articles/serializers.py

```diff
@@ -1,7 +1,7 @@
 import re
 
 from rest_framework import serializers
-from .models import Article
+from .models import Article, ArticleRating
```

Comment:
ArticleRating is not being used in this file

Reply:
Let me work on that.